en/docs/develop/session-mgt-rest-api.md

```diff
@@ -19,6 +19,16 @@ template: templates/swagger.html
        4. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2
          IS. 
          
+!!! Note
+    
+    - From IS5.12 version onwards, We revoke the access tokens  mapped with the session via REST API.
+    - This is applicable for authorization code grant, implicit flow, refresh token grant and hybrid flow. 
+    - If the same accesstoken is used my multiple sessions, then if one session is terminated via API, 
+      the other sessions may loose the tokens. So it is recommended to use sso-session binding if you want to have the 
+      functionality to revoke the accesstoken when the session is termianted via REST API.
+    - WSO2 Identity Server sends SAML and OIDC back-channel logouts also when session is termianted via API. To support 
+      this, the application should be configured back-channel logouts.
```

Comment:
```suggestion
    - WSO2 Identity Server sends SAML and OIDC back-channel logouts also when session is terminated via API. To support 
      this, the application should be configured to enable back-channel logout.
```

Reply:
"sends SAML and OIDC back-channel logouts" --> it is not very clear what is meant by this. What are we trying to say here?